designs/administrator/future-openshift/saved-searches/index.md

```diff
@@ -0,0 +1,46 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Saved filters
+
+The Search page will be updated to allow users to save searches.
+
+## General flow
+
+![1](img/saved1.png)
+* If no filters have been saved, the **Saved filters** dropdown will have a small empty state.
+
+![2](img/saved2.png)
+* Users can save filters by clicking the **Save filter** button at the end of the filter chip row.
+
+![3](img/saved3.png)
```

Comment:
nit: it could be nice to call this Save search filter
same for the description text: Save the current search filter...

Reply:
Hm, if you don't feel strongly on this, I may prefer to leave it as "Save filter". My reasoning is that the page is called Search so technically you are saving the "Search filter" but may be confusing because "Save search" sounds like a similar function. In that way I wonder if "Save search filter" clarifies or confuses. I imagine we wouldn't name this "Save Pod filter" if it appeared on the Pods page. What do you think?